tempesta_fw/vhost.h

```diff
@@ -144,10 +151,7 @@ enum {
  * @loc_dflt	- Default policy.
  * @vhost_dflt	- Pointer to default virtual host with global policies.
  * @hdrs_pool	- Modification headers allocation pool for vhost's policies.
- * @frang_gconf	- Global frang configuration. Applicable only for 'default'
- *		  vhost and NULL for others. Provides frang configuration
- *		  options used before request is parsed and assigned to any
- *		  vhost.
```

Comment:
`frang_gconf` field is still present. Perhaps, it's worth to keep its description as well?